plugins/modules/net_tools/nmcli.py

```diff
@@ -74,6 +74,16 @@
             - The IPv4 gateway for this interface.
             - Use the format C(192.0.2.1).
         type: str
+    routes4:
+        description:
+            - The list of ipv4 routes.
+            - Use the format '192.0.3.0/24 192.0.2.1,192.0.4.0/24 192.0.2.1'
+        type: str
```

Comment:
If it is a list, declare it as a list and transform it to the correct format.
```suggestion
        type: list
        elements: str
        version_added: 1.3.0
```

Reply:
Thanks, solved !